config/manager/kustomization.yaml

```diff
@@ -4,5 +4,5 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: ashimagarg/csi
-  newTag: "53"
+  newName: bhagyak1/ibm-object-csi-driver-operator
+  newTag: m1402
```

Comment:
Please avoid personal registry and tags here.

Reply:
yes removed it.